.golangci.yml

```diff
@@ -19,6 +19,9 @@ linters-settings:
       - fieldalignment
   misspell:
     locale: US
+  staticcheck:
+    checks: ["all", "-SA1019"]
```

Comment:
I fixed everything I could, then disabled while we still depend on `openpgp` here -- see https://github.com/golang/go/issues/44226.